devops/openshift/api-deploy.yml

```diff
@@ -213,6 +213,8 @@ objects:
                   value: ${BYPASS_CRA_INCOME_VERIFICATION}
                 - name: BYPASS_APPLICATION_SUBMIT_VALIDATIONS
                   value: ${BYPASS_APPLICATION_SUBMIT_VALIDATIONS}
+                - name: SWAGGER_ENABLED
```

Comment:
The parameters must be added to the _parameters_ section at the end of the file.
Please search by BYPASS_APPLICATION_SUBMIT_VALIDATIONS to see an example.